examples/example-event-handler/dist/ExampleSubgraph/ExampleSubgraph.ts

```diff
@@ -463,6 +464,41 @@ class Value {
   kind: ValueKind
   data: ValuePayload
 
+  toAddress(): Address {
+    assert(this.kind == ValueKind.BYTES, 'EthereumValue is not an address.')
```

Comment:
None of these are `EthereumValue`s, they are all `Value`s. Can you change the assertions?